cli/flags.rs

```diff
@@ -590,6 +602,14 @@ pub fn flags_from_vec(
       argv.extend(vec![source_file.to_string(), out_file.to_string()]);
       DenoSubcommand::Bundle
     }
+    ("completions", Some(_completions_match)) => {
+      create_cli_app().gen_completions_to(
+        "deno",
+        Shell::Bash,
+        &mut std::io::stdout(),
+      );
+      std::process::exit(1);
```

Comment:
Why exit(1) ?

Reply:
No reason, I was prototyping late at night and just copy pasted from other place... I'll ping you when it's ready for review